src/index.js

```diff
@@ -245,31 +201,30 @@ class Main {
     };
 
     // Curent element of focusList in focus
-    this.focusList = [this.lambdasTable, this.eventBridgeTree, this.map.map];
+    this.focusList = [this.resourceTable.table, this.eventBridgeTree, this.map.map];
     this.returnFocus();
     this.isModalOpen = false;
 
     // Dictionary to store previous submissions for each event bus
     this.previousSubmittedEvent = {};
     // Dictionary to store previous submissions for each lambda function
     this.previousLambdaPayload = {};
-    this.lambdasTable.rows.on("select", (item) => {
-      [this.funcName] = item.data;
-      this.fullFuncName = `${program.stackName}-${this.funcName}`;
-    });
   }
 
   setKeypresses() {
     screen.key(["d"], () => {
```

Comment:
Can we set the keys on the resource table, look at how its done in the map, can maybe get rd of the focus index as well